includes/misc/apache/update.conf

```diff
@@ -0,0 +1,12 @@
+<VirtualHost *:80>
+	DocumentRoot /opt/updateui
```

Comment:
This looks odd. It does not seem o be part of this change. Can you please explain the purpose?

Reply:
yea these changes are under progress as we need apache instead of nginx, will delete this apache folder for now